ppb_vector/vector2.py

```diff
@@ -198,42 +201,42 @@ def __add__(self: Vector, other: VectorLike) -> Vector:
         """Add two vectors.
 
         :param other: A :py:class:`Vector2` or a vector-like.
-          For a description of vector-likes, see :py:func:`convert`.
+          For a description of vector-likes, see :py:func:`__new__`.
 
         >>> Vector2(1, 0) + (0, 1)
         Vector2(1.0, 1.0)
         """
         rtype = _find_lowest_vector(type(other), type(self))
         try:
-            other = Vector2.convert(other)
+            otherx, othery = Vector2._unpack(other)
```

Comment:
```suggestion
            other_x, other_y = Vector2._unpack(other)
```

I think that would be easier to read; of course, that needs to be fixed thorough the code, but that's nothing a search-and-replace cannot handle. I don't have much opinions on snake_case vs. camelCase, but I think the former is customary in Python?)